.scripts/deploy.sh

```diff
@@ -4,21 +4,30 @@
 # NOTE: Running build.sh and then deploy.sh is not finding the public
 # folder which is generated after the build command (hugo).
 git submodule update --init
+
+# Find PR number and determine where to deploy to with GitHub Actions
+PR_NUMBER="$(echo $REF | cut -d'/' -f3)"
+DOMAIN_NOHTTP="numpy-${PR_NUMBER}-numpy.org-newsite.surge.sh"
```

Comment:
Should we update this name to remove `newsite`? We could probably remove the `numpy.org` before it as well, if you'd like.

Reply:
Yes, I just wanted to check whether it worked before changing the name. It doesn't at the moment, and I'm not sure if the name has anything to do with it. @Shekharrajak any thoughts?

Reply:
URL can be anything. It is not a problem.

Reply:
I think the issue is with Hugo and how its relative & absolute URL's work. I don't think there's a deploy script/surge issue (that's my intuition, anyway).
@Shekharrajak/@rgommers, I had updated urls to relative URLs, but I don't think that worked. Any thoughts/suggestions here?

Reply:
@joelachance I'm fairly sure that's unrelated to this. If it doesn't deploy to Surge in the first place because the secret tokens don't show up, we never get to the point where absUrl/relUrl matters

Reply:
Ah, I thought it had been loading. You're right.
I think it may have been working previously, but it takes a *very* long time to load sometimes...